### PR TITLE
[homematic] Update extra-descriptions_de.properties

### PR DESCRIPTION
**Pull request by BOFH90**
HmIP-MOD-TM added to List
Devices works flawlessly but had no description.

### CODE REVIEW

File: bundles/org.openhab.binding.homematic/src/main/resources/homematic/extra-descriptions_de.properties [hunk 1]
```diff
@@ -19,104 +19,105 @@ BC-SC-Rd-WM-2=MAX! Fensterkontakt
...
 HmIP-BWTH=Wandthermostat mit Schaltausgang
 HmIP-BWTH24=Wandthermostat mit Schaltausgang
-HmIP-PCBS-BAT=Schaltplatine f�r Batteriebetrieb
+HmIP-PCBS-BAT=Schaltplatine für Batteriebetrieb
 HmIP-FROLL=Homematic IP Rolladenaktor
-HmIP-BROLL=Homematic IP Rolladenaktor f�r Markenschalter
-HmIP-BRC2=Homematic IP Wandtaster f�r Markenschalter - 2-fach
+HmIP-BROLL=Homematic IP Rolladenaktor für Markenschalter
+HmIP-BRC2=Homematic IP Wandtaster für Markenschalter - 2-fach
+HmIP-MOD-Homematic IP Tormatic Modul
```
Comment (martinvw) [inline bundles/org.openhab.binding.homematic/src/main/resources/homematic/extra-descriptions_de.properties]: What are your changes? to this file only this? It seems that the equals misses here.

Comment (BOFH90) [reply bundles/org.openhab.binding.homematic/src/main/resources/homematic/extra-descriptions_de.properties]: I added the "HmIP-MOD-Homematic IP Tormatic Modul". Its the only changed line. I don´t know why the GitHub-Web-Editor marked the other Lines as changed.

Comment (martinvw) [reply bundles/org.openhab.binding.homematic/src/main/resources/homematic/extra-descriptions_de.properties]: Do you agree with the DCO? See https://community.openhab.org/t/dco-check-signing-off-with-github-web-editor-explanation/83330

If so then I can recreate your commit and fix it using a normal PR.

Comment (BOFH90) [reply bundles/org.openhab.binding.homematic/src/main/resources/homematic/extra-descriptions_de.properties]: Yes, this would be great

**Pull request closed without merging**